### ISSUE TITLE
Add Tooltip to Histogram Panels

### ISSUE THREAD

**Issue by danwalkeruk** [grafana/grafana#79375]
**Why is this needed**:

Tooltips are missing on Histogram panels, whereas they exist on almost every other type of panel. I'm not sure why they weren't added, it seems like an oversight.

This is a continuation of [this](https://github.com/grafana/grafana/discussions/47848#discussioncomment-7828455) discussion which was closed due to requests being moved to GitHub Issues

**What would you like to be added**:

Tooltips to histograms.

**Who is this feature for?**

Everyone.

**Comment by usmangt** [comment]
Hi @danwalkeruk 

Thanks for opening this issue. This sounds like a feature request.

I will forward this to our Engineers so that they can have a look and might be able to tell more details about it.

**Comment by WB-FSE** [comment]
Up!

**Comment by justinsmalley** [comment]
I've been using the histogram mode from the old graph panel because it has this feature. That panel will unfortunately be made [permanently unavailable](https://grafana.com/docs/grafana/latest/developers/angular_deprecation/) with the release of Grafana 11 in approximately Q2 2024.

**Comment by leeoniya** [comment]
>  I'm not sure why they weren't added, it seems like an oversight.

the main reason is that each panel had its own bespoke and inconsistent tooltip code, and we did not want to make the situation worse, which would increase the refactoring burden when we eventually arrived at a uniform model. [1]

it took a while to get here, but we're nearing the end, so it should now be much easier to implement.

counter-intuitively, the current underlying viz code is much closer to TimeSeries and Trend that it is to BarChart, so the initial tooltip interaction model may be closer to the former than the latter. but we can refine this after _something_ exists.

[1] https://github.com/grafana/grafana/issues/68238
